packages/core/src/input/input.element.scss

```diff
@@ -14,6 +14,8 @@
   --border-radius: 0;
   --outline: none;
   --padding: 0 #{$cds-global-space-4};
+  --padding: #{$cds-global-space-2} #{$cds-global-space-4} 0 #{$cds-global-space-4}; // 1px top for line height alignment with label
```

Comment:
Can we update the comment here or remove it? it says `1px` but it's actually `2px`. Maybe it can say `added padding to the top for line-height adjustment with label`?

Reply:
done.